arkouda/dataframe.py

```diff
@@ -440,8 +491,8 @@ def _repr_html_(self):
         """
         Return html-formatted version of the dataframe.
         """
+        prt = self._get_head_tail_server()
 
-        prt = self._get_head_tail()
```

Comment:
I really love that you benchmarked both the old and new versions of this function! Although the single-node results look really promising, I'm wondering if we should look at how those benchmarks compare on the 16-node cases before we switch the implementation. We could merge this PR while still using `self._get_head_tail()`, and then if the benchmarks look good, it would be a one-line PR to change the implementation. On the other hand, it would save us some trouble if there's a hidden gotcha in the multi-locale case.

Reply:
This makes sense to me. I update `_repr_html_()` and `__repr__()` to use `self._get_head_tail()`. The benchmark has been updated to remove `_repr_html_()` from the operations to test and will now only directly compare `_get_head_tail()` and `get_head_tail_server()`